goset_test.go

```diff
@@ -165,7 +165,7 @@ func TestSet_IsDisjoint(t *testing.T) {
 func TestSet_String(t *testing.T) {
 	s := NewSet[string]("a", "b")
 	str := fmt.Sprintf("%v", s)
-	possibleOutputs := []string{"Set[string]{a b}", "Set{b a}"}
+	possibleOutputs := []string{"Set[string]{a b}", "Set[string]{b a}", "Set{b a}"}
```

Comment:
Actually, `Set{b a}` is not a possible output, it's my mistake, you can delete it from the slice :)

Reply:
Woohoo! Done.